src/go/rpk/pkg/cli/cmd/cluster/license/info.go

```diff
@@ -47,6 +48,10 @@ func newInfoCommand(fs afero.Fs) *cobra.Command {
 				} else {
 					printLicenseInfo(info.Properties)
 				}
+				if info.Properties.Expires < 0 {
+					fmt.Fprintln(os.Stderr, "warning: your license is expired")
+					return
+				}
```

Comment:
Can we exit 1 here?

Reply:
Ok, wdyt of changing the error then:

```
out.Die("Error: your license is expired")
```

And add a command description to say that we exit 1 when the license is expired?

Reply:
What about adding a field, `Expired`, to the json and to the plaintext only when the license is already expired? this will ensure we don't have invalid json output?

Perhaps in that case we won't need to exit 1. I mostly just think we should avoid invalid json for the json format.